src/Applications/NCEP_Etc/NCEP_enkf/scripts/gmao/etc/HISTAENS.rc.tmpl

```diff
@@ -863,3 +865,47 @@ COLLECTIONS: 'bkg.eta'
                         'TDEL'             , 'SURFACE'     , 'tdel'       ,
                         'Z0'               , 'SURFACE'     , 'z0m'        ,
                         ::
+
+#######################################################################
+#                predictor forcing files for land da coupling 
+#######################################################################
+
+  bkg.tavg1_2d_lfo.format:     'CFIO' ,
+  bkg.tavg1_2d_lfo.descr:      '2d,1-Hourly,Time-Averaged,Single-Level,Land Surface Forcings' ,
+# bkg.tavg1_2d_lfo.nbits:      10 ,
+  bkg.tavg1_2d_lfo.template:   '%y4%m2%d2_%h2%n2z.>>>MEMBER<<<.>>>NCSUFFIX<<<' ,
+  bkg.tavg1_2d_lfo.mode:       'time-averaged' ,
+  bkg.tavg1_2d_lfo.frequency:  010000 ,
+  bkg.tavg1_2d_lfo.duration:   010000 ,
+  bkg.tavg1_2d_lfo.ref_date:   >>>IOBBKGD<<< ,
+  bkg.tavg1_2d_lfo.ref_time:   >>>IOBBKGT<<< ,
+  bkg.tavg1_2d_lfo.end_date:   >>>IOEDATE<<< ,
+  bkg.tavg1_2d_lfo.end_time:   >>>IOETIME<<< ,
+  bkg.tavg1_2d_lfo.fields:     'SLRSF'   , 'SOLAR'    , 'SWGDN'    ,
+                                'SWLAND'  , 'SURFACE'  ,
```

Comment:
I'm pretty sure we do not need the **net** SW (SWLAND) in the lfo tavg file. It's a remnant from way back when.  I recommend removing the SWLAND variable and trying a single cycle without it to verify

Reply:
yes, SWLAND should be removed from both the deterministic and ensemble HISTORY files